GUI/Main/MainWait.cs

```diff
@@ -87,5 +87,14 @@ public void Wait_OnCancel()
                 cancelCallback();
             }
         }
+
+        public void Wait_OnOk()
+        {
+            ManageMods.UpdateModsList(null);
```

Comment:
In principle, this tab is supposed to be generic and usable by any task that needs to show progress, for example repository updates, auto-updating CKAN, and downloading files from the right click menu, so it shouldn't make assumptions about which tasks it is displaying. Is there a place more specific to the mod list where we could put the `UpdateModsList` call?

Reply:
Hm, maybe we can make that another callback (like `Wait_OnCancel()` above), and (un)register `UpdateModsList()` to it during `PostInstallMods` or somewhere else. Need to try that out.